.github/CONTRIBUTING.md

```diff
@@ -6,16 +6,16 @@
 
 Hi! Thanks for your interest in contributing to the GitHub CLI!
 
-Given that this project is very early and still in beta, we're only accepting pull requests for bug fixes right now. We'd love to
-hear about ideas for new features as issues, though!
+We accept pull requests for bug fixes and features where we've discussed the approach in an issue and given the go-ahead for a community member to work on it. We'd also love to hear about ideas for new features as issues.
 
 Please do:
 
 * open an issue if things aren't working as expected
 * open an issue to propose a significant change
 * open a PR to fix a bug
```

Comment:
should we also qualify this one, too? like "as long as we've agreed on the solution together"

Reply:
or just delete this one in light of the next bullet point?

Reply:
I actually think the two use cases are slightly different. They can go ahead and fix a bug if they find an obvious one, but any new functionality should have discussion and agreement in an issue first.

Reply:
That’s fair! l’m worries that folks will introduce new behavior while fixing a bug — but I think it’s an unfounded worry. I’m down to clarify later if the guidelines get misconstrued. Looks good to me!!